include/sound/sof.h

```diff
@@ -47,6 +47,13 @@ enum sof_dsp_power_states {
 	SOF_DSP_PM_D3,
 };
 
+/* Definitions for multiple IPCs */
+enum sof_ipc_types {
```

Comment:
I think this should not be plural: `enum sof_ipc_type`

Reply:
done